src/backend/utils/ping-discord.ts

```diff
@@ -0,0 +1,15 @@
+export const sendToDiscord = async (message: string) => {
+  const req = await fetch(
+    "https://discord.com/api/webhooks/888615431617785876/nNcSPs4yZZVqaGc-ohEO0Hw55tqMt7ywntb6R6AOzZfSylw7O2TeGk7lXhMDecKBbPjZ",
```

Comment:
Shouldn't this be a secret or something? 😅

Reply:
I thought I left a comment above saying such lol my bad, can generate a new one and kill this link

Reply:
Updated accordingly :)